weave/trace_server/migrations/003_add_display_name.down.sql

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
I think you are going to need to bump the migration numbers!

Reply:
oops was slightly stale @tssweeney